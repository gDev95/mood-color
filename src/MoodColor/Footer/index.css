.footer-wrapper {
    display: flex;
    justify-content: center;
    flex-direction: column;
    height:15vh;
    width:100%;
    text-align: center;
    color:#FFF;
    background-color:#000;
}
.footer-text {
    margin:0;
    font-family: 'Kavivanar', cursive;
}
.footer-icon {
    margin:5px;
    text-decoration: none;
    color:#FFF;
}
.footer-icon:visited{
    text-decoration: none;
}
.footer-icon:focus {
    text-decoration: none;
}