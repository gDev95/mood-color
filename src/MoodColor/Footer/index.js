import React from 'react'
import './index.css'
import FontAwesome from 'react-fontawesome'
import { Link, Router } from 'react-router-dom'
const Footer = () => (
	<div className='footer-wrapper'>
		<div className='footer-text'> 
			<a className='footer-icon' target='_blank' href='https://github.com/gDev95'>
				<FontAwesome  size='2x' name='github' />
			</a>
			<a className='footer-icon' target='_blank' href='https://www.linkedin.com/in/paul-meyer-046798a8'>
				<FontAwesome className='footer-icon' size='2x' name='linkedin' />
			</a>
		</div>
	</div>    
)
export default Footer