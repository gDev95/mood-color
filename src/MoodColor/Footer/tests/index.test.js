import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import Footer from '../index.js'


describe('Footer component',() =>{
	const component = shallow(
		<Footer />
	)
	test('renders properly', () => {
		const tree = toJSON(component)
		expect(tree).toMatchSnapshot()
	})      
})