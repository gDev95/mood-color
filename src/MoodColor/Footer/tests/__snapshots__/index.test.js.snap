// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders properly 1`] = `
<div
  className="footer-wrapper"
>
  <div
    className="footer-text"
  >
    <a
      className="footer-icon"
      href="https://github.com/gDev95"
      target="_blank"
    >
      <FontAwesome
        name="github"
        size="2x"
      />
    </a>
    <a
      className="footer-icon"
      href="https://www.linkedin.com/in/paul-meyer-046798a8"
      target="_blank"
    >
      <FontAwesome
        className="footer-icon"
        name="linkedin"
        size="2x"
      />
    </a>
  </div>
</div>
`;
